components/NavigationLogo.jsx

```diff
@@ -1,17 +1,15 @@
 import { ButtonBase } from '@mui/material';
+import Image from 'next/image';
```

Comment:
This is another rule I think I will add to PR checklist, all images should be using this. I think there are several instances around the repo where we are doing <img> and some where we are doing it properly, like this.

Reply:
I saw Next is no longer supporting img tags. Put on our ever-growing list